src/helpers/validators.ts

```diff
@@ -0,0 +1,27 @@
+const emailRegex = /\S+@\S+\.\S+/;
+const isValidEmail = (testEmail : string) => emailRegex.test(testEmail);
+
+const strongRegex = /^(?=.*[!@.#$%^&?*])(?=.*[a-z]).{8,}$/;
```

Comment:
it's good practice to leave a link from where you found these regex's as a comment